facade/src/main/java/com/redhat/lightblue/migrator/facade/TimeoutConfiguration.java

```diff
@@ -118,7 +118,17 @@ public long getMS(String methodName, FacadeOperation op, Type type) {
                 log.debug("{} config not found for bean {} using global timeout", type, beanName);
             }
 
-            timeout = defaultTimeoutMS;
+            switch (type) {
+                case timeout: {
+                    timeout = defaultTimeoutMS; break;
+                }
+                case slowwarning: {
+                    timeout = 2 * defaultTimeoutMS; break;
```

Comment:
I see what you mean. Moved that to the constructor and changed so that defaultSlowwarnMS = defaultTimeoutMS. I didn't want both timeout and slow call warns to be logged for the same calls, but I realized this can be confusing.